custom_components/tahoma/tahoma_entity.py

```diff
@@ -81,9 +84,6 @@ def device_state_attributes(self) -> Dict[str, Any]:
         """Return the state attributes of the device."""
         attr = {}
 
-        if self.has_state(CORE_RSSI_LEVEL_STATE):
-            attr[ATTR_RSSI_LEVEL] = self.select_state(CORE_RSSI_LEVEL_STATE)
-
         if self.has_state(CORE_BATTERY_STATE):
```

Comment:
This will be a harder one to migrate? Are there situations where we have two battery states? One with the number and one with just 'normal / low' etc? If that is never the case, we could migrate this one as well.